src/services/bindService.js

```diff
@@ -11,7 +11,8 @@ angular.module("openshiftCommonServices")
     // The secret key this service uses for the parameters JSON blob when binding.
     var PARAMETERS_SECRET_KEY = 'parameters';
 
-    var bindingResource = APIService.getPreferredVersion('servicebindings');
+    var serviceBindingVersion = APIService.getPreferredVersion('servicebindings');
```

Comment:
serviceBindingsVersion (plural) to be consistent

Reply:
Updated